src/main/scala/diplomacy/Nodes.scala

```diff
@@ -259,27 +733,157 @@ trait OutwardNode[DO, UO, BO <: Data] extends BaseNode
+  *                  │            │                 │                                                                      │                     │ │
+  *                  │            │                 │                                                           [[MixedNode.outer.edgeO]]        │ │
+  *                  │            │                 │                                                           (based on protocol)              │ │
+  *                  │            │                 │                                                                      │                     │ │
+  *                  │            │                 │                             ┌────────────────────────────────────────┤                     │ │
+  *                  │            │                 │                             │                                        │                     │ │
+  *                  │            │                 │                             │                                        │                     │ │
+  *                  │            │                 │                             │                                        │                     │ │
+  * (immobilize after elaboration)│                 ↓                             │                                        │                     │ │
+  *     [[OutwardNode.oBindings]]─┘      [[MixedNode.oDirectPorts]]───→[[MixedNode.oPorts]]                   [[OutwardNode.doParams]]           │ │
+  *                  ↑               (inward port from [[OutwardNode]])           │                                        │                     │ │
+  *                  │                  ┌─────────────────────────────────────────┤                                        │                     │ │
+  *                  │                  │                                         │                                        │                     │ │
+  *                  │                  │                                         │                                        │                     │ │
+  *     [[OutwardNode.accPO]]           │                                         ↓                                        │                     │ │
+  *   (binding node when elaboration)   │ [[InwardNode.diParams]]─────→[[MixedNode.mapParamsD]]────────────────────────────┘                     │ │
+  *                                     │             ↑                                                                                          │ │
+  *                                     │             └──────────────────────────────────────────────────────────────────────────────────────────┘ │
+  *                                     └──────────────────────────────────────────────────────────────────────────────────────────────────────────┘
+  */
```

Comment:
This diagram has the Scala unidoc issue, `MixedNode.xxx` cannot be resolved except `inner` and `outer` `val`. @jackkoenig do you any idea?

Reply:
It's a bit too wide to render all that well, but I pushed a commit that improves it a lot.